packages/clerk-js/src/ui/utils/useFormControl.ts

```diff
@@ -14,6 +14,7 @@ type Options = {
   placeholder?: string | LocalizationKey;
   options?: SelectOption[];
   checked?: boolean;
+  initialCheckboxState?: boolean;
```

Comment:
🙃 
As per React naming convention, `checked` vs `defaultChecked`

```suggestion
  defaultChecked?: boolean;
```

Reply:
@raptisj  If we introduce defaultChecked I think we can removed check from options right ? 

options?.checked was meant to populate the initial value of the checkbox